npm-intro/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "npm-intro",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "jquery": "^3.6.0",
+    "jQuery": "^1.7.4"
```

Comment:
If you install the wrong package you can use `npm uninstall package-name` to remove it